event_manager/lib/event_manager_test.rb

```diff
@@ -0,0 +1,37 @@
+gem 'minitest', '~> 5.0'
+require 'minitest/autorun'
+require 'minitest/pride'
+require_relative 'event_manager'
+
+class EventManagerTest < Minitest::Test
+  # HomePhone Tests
+  def test_phone_more_than_11_digits
+    assert_equal false, validate_phone("888888888882")
+  end
+
+  def test_phone_less_than_10_digits
+    assert_equal false, validate_phone("6743709")
+  end
+
+  def test_phone_equals_10_digits
+    assert_equal true, validate_phone("8765789065")
+  end
+
+  def test_phone_equals_11_digits_firstis_1
+    assert_equal "8765789065", validate_phone("18765789065")
```

Comment:
I don't like it that way. Now we have one method that returns either `bool` or `string`; it's bad practice. 

I think we need two methods here - one of them (`sanitize_phone`) handling striping first digit if it's `1` and number has 11 digits, otherwise returning original number. Other method `validate_phone` should take output from `sanitize_phone` and return `true` or `false`.